package.json

```diff
@@ -1,44 +1,46 @@
 {
-  "name": "di-ipv-cri-toy-api",
-  "version": "1.0.0",
+  "name": "ipv-cri-toy-api",
   "description": "API for the toy credential issuer",
   "main": "index.js",
   "scripts": {
+    "pre-commit": "pre-commit run --all-files --show-diff-on-failure",
     "lint:prettier": "prettier --check lambdas",
     "lint:prettier-fix": "npx prettier --write .",
     "lint:eslint": "eslint .",
-    "lint": "npm run lint:prettier && npm run lint:eslint",
+    "lint": "npm run lint:eslint",
+    "lint:fix": "npm run lint:eslint -- --fix",
     "compile": "tsc",
-    "unit": "jest --config=jest.config.ts",
-    "test": "npm run compile && npm run unit",
+    "unit": "jest --silent",
+    "test": "npm run unit",
+    "test:coverage": "npm run unit -- --coverage",
     "test:browser": "cucumber-js --config tests/browser/cucumber.js"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/alphagov/di-ipv-cri-toy-api.git"
+    "url": "git+https://github.com/govuk-one-login/ipv-cri-toy-api.git"
   },
   "author": "alphagov",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/alphagov/di-ipv-cri-toy-api/issues"
+    "url": "https://github.com/govuk-one-login/ipv-cri-toy-api/issues"
   },
-  "homepage": "https://github.com/alphagov/di-ipv-cri-toy-api#readme",
+  "homepage": "https://github.com/govuk-one-login/ipv-cri-toy-front/#readme",
   "devDependencies": {
     "@cucumber/cucumber": "8.6.0",
     "@types/aws-lambda": "^8.10.114",
     "@types/jest": "^29.5.0",
-    "@typescript-eslint/eslint-plugin": "5.10.2",
-    "@typescript-eslint/parser": "5.10.2",
+    "@typescript-eslint/eslint-plugin": "^6.7.5",
```

Comment:
Please install without the carets, as dependabot covers the ability to update to minor versions.